Dockerfile

```diff
@@ -0,0 +1,14 @@
+# ----- EIB Builder Image -----
+FROM golang:1.21
+
+WORKDIR /src
+COPY . ./
+RUN go build ./cmd/eib
+
+
+# ----- Deliverable Image -----
+FROM registry.suse.com/bci/bci-base:15.4
```

Comment:
I'm not sure which of the BCI images is the most suitable for our use case. AFAICS, we have the following options:
- bci-base
- bci-minimal
- bci-micro

The size difference of the final image between `bci-base` and `bci-micro` exceeds 100 MB so this is definitely something we'd want to investigate later. I'm good with starting with `bci-base` though.

I believe we should be using SP5 though (i.e. 15.5).

Reply:
eib seems to be a nice static binary, so bci-micro is the right one

Reply:
Tested a change to micro (and the correct version, not sure why I used 15.4) and it looks good, I'll make the change.